packages/utils/src/lib/reports/__snapshots__/log-stdout-summary.integration.test.ts.snap

```diff
@@ -161,7 +158,159 @@ Lighthouse audits
 ● Cumulative Layout Shift                                             0
 ● Total Blocking Time                                                 0 ms
 
+Made with ❤ by code-pushup.dev"
+`;
 
-Made with ❤ by code-pushup.dev
-"
+exports[`logStdoutSummary > should contain all sections when using the fixture report 1`] = `
```

Comment:
The duplicate snapshots should be removed.

```suggestion
exports[`logStdoutSummary > should contain all sections when using the fixture report 1`] = `
```

Reply:
removed duplicated snapshot